go.mod

```diff
@@ -169,10 +169,10 @@ require (
 
 replace (
 	// Knative forks.
-	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20220926135101-4a89adaca7d1
-	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20220504141528-c59955f23883
-	knative.dev/eventing-kafka-broker => github.com/openshift-knative/eventing-kafka-broker v0.25.1-0.20220927171706-e0d47343c338
-	knative.dev/serving => github.com/openshift/knative-serving v0.10.1-0.20220926120634-72a6f7ea6cfb
+	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20221109093310-866f17f1f1f9
+	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20221109100552-26eea5293bca
+	knative.dev/eventing-kafka-broker => github.com/openshift-knative/eventing-kafka-broker v0.25.1-0.20221109150735-a948d166b5ee
+	knative.dev/serving => github.com/openshift/knative-serving v0.10.1-0.20221104105313-b02fdc605c48
```

Comment:
Cool as discussed in slack we should have latest.
Serving last commit on 1.5 is `b02fdc605c4811b418dcb7e53547e17d4d688631` so seems right to me.